documentation/autoinstall-reference.md

```diff
@@ -504,7 +504,7 @@ Supported values are:
 ### shutdown
 
 **type:** string (enumeration)
-**default:** do nothing
```

Comment:
That reminds me that we actually want a do-nothing power mode (not this PR of course).

Reply:
:+1: that would be good to have!